src/angular-app/bellows/apps/userprofile/user-profile-app.component.html

```diff
@@ -68,34 +68,6 @@
                                 </div>
                             </div>
                         </div>
-                        <div class="form-group row">
-                            <div class="card bg-light col-md-9 col-lg-7">
-                                <div class="card-body">
-                                    <div class="form-group row">
-                                        <label class="col-form-label col-md-4 text-md-right" for="mobile_phone">Mobile Phone Number</label>
-                                        <div class="col-md-8">
-                                            <input class="form-control" type="tel" data-pui-intl-tel-input id="mobile_phone" placeholder="###-####" data-ng-model="$ctrl.user.mobile_phone">
```

Comment:
removing the intl-tel-input will be a big win!  We can do a separate PR to remove this dependency and also JQuery, on which this old piece depends.

Reply:
went ahead and took care of this while getting rid of the intl-tel-input, see [7bdbe26](https://github.com/sillsdev/web-languageforge/pull/1611/commits/7bdbe2635f75a8982f61f849f6a45d22bd41426e)

Reply:
great thanks!